pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -1301,4 +1308,25 @@ public static class NullValuePlaceHolder {
     public static final byte[][] BYTES_ARRAY = new byte[0][];
     public static final Object MAP = Collections.emptyMap();
   }
+
+  public static class CursorConfigs {
+    public static final String DEFAULT_RESPONSE_STORE_TYPE = "file";
+    public static final String DEFAULT_RESPONSE_SERDE = "json";
+    public static final int MAX_CURSOR_FETCH_ROWS = 100000;
+    public static final int DEFAULT_CURSOR_FETCH_ROWS = 10000;
+    public static final String DEFAULT_RESULTS_EXPIRATION_INTERVAL = "1h"; // 1 hour.
+    public static final String PREFIX_OF_CONFIG_OF_CURSOR = "pinot.broker.cursor";
+    public static final String PREFIX_OF_CONFIG_OF_RESPONSE_STORE = "pinot.broker.cursor.response.store";
+    public static final String RESPONSE_STORE_TYPE = "type";
+    public static final String RESPONSE_STORE_SERDE = "serde";
+    public static final String CURSOR_FETCH_ROWS = PREFIX_OF_CONFIG_OF_CURSOR + ".fetch.rows";
+    public static final String RESULTS_EXPIRATION_INTERVAL = PREFIX_OF_CONFIG_OF_RESPONSE_STORE + ".expiration";
+
+    public static final String RESPONSE_STORE_CLEANER_FREQUENCY_PERIOD =
+        "controller.cluster.response.store.cleaner.frequencyPeriod";
+    public static final String DEFAULT_RESPONSE_STORE_CLEANER_FREQUENCY_PERIOD = "1h";
+    public static final String RESPONSE_STORE_CLEANER_INITIAL_DELAY =
+        "controller.cluster.response.store.cleaner.initialDelay";
```

Comment:
Why does this need to be configurable? The default random initial delay should be good enough right? Pinot already has a gazillion different configurations and it'd be nice to avoid adding unnecessary new ones IMO. It leads to a super messy user experience when there are so many irrelevant configs to understand and learn about.

Reply:
I dont know the history why all periodic tasks have this config param. Agree with the sentiment re: too many config params. However lets change any common patterns outside this PR.

Reply:
Then this will be the only periodic task without a parameter to configure initial delay.

Reply:
Not sure why that matters for this task though? Maybe it's more relevant for the other tasks, haven't taken a look.

Reply:
Not specific. Basically dont want a herd of periodic tasks running regularly every hour etc. So start each task in a 5 min range.